### ISSUE TITLE
Support DKIM l= tag (body length)

### ISSUE THREAD

**Issue by falstaff84** [vstakhov/rmilter#61]
The l= tag is useful for mailing lists and message handling system which add content to the end... The dkim_sign function has a parameter for body length, I guess if the whole body length is passed, the function should add the l= tag and sign the complete mail.

**Comment by vstakhov** [comment]
Nope, dkim context is obtained when the whole length of message is yet unknown. Anyway, I don't see any useful application of `l=` in the context of rmilter.

**Comment by falstaff84** [comment]
Well, when sending emails to mailing list, would be worthwhile I guess? Currently my emails fail DKIM signature when I receive them back through the mailing list... 

I there another way to archive this with a rmilter/rspamd (maybe second OpenDKIM milter?)
